### PR TITLE
[Backport ADF-590] [Tabular Import]: The metadata values in the CSV are not imported when in field "Alias" user uses capital letters

### PR DESCRIPTION
**Pull request by habemuscode**
**Ticket**

https://oat-sa.atlassian.net/browse/ADF-605

### CODE REVIEW

File: model/import/Parser/CsvLineConverter.php [hunk 1]
```diff
@@ -26,13 +26,16 @@
...
 
 class CsvLineConverter extends ConfigurableService
 {
-    public function convert(array $line, TemplateInterface $template, $validationReport = null): ItemInterface
-    {
+    public function convert(
+        array $line,
+        TemplateInterface $template,
+        AggregatedValidationException $exception = null
+    ): ItemInterface {
```
Comment (gabrielfs7) [inline model/import/Parser/CsvLineConverter.php:38]: We need to revert it

File: model/import/Parser/CsvLineConverter.php [hunk 2]
```diff
@@ -43,7 +46,7 @@ public function convert(array $line, TemplateInterface $template, $validationRep
                 $parser = $this->getServiceLocator()->get($rules['parser']);
                 $parsed[$columnName] = $parser->parse($line, $rules, ['columnName' => $columnName]);
             } else {
-                $isInvalid = $validationReport && $this->hasValidationIssues($columnName, $validationReport);
+                $isInvalid = $exception && $exception->hasColumnWarning($columnName);
```
Comment (gabrielfs7) [inline model/import/Parser/CsvLineConverter.php:49]: We need to revert it

File: model/import/Parser/CsvLineConverter.php [hunk 3]
```diff
@@ -52,29 +55,24 @@ public function convert(array $line, TemplateInterface $template, $validationRep
         return new CsvItem(
             $parsed['name'],
             $parsed['question'],
-            $this->normalizeShuffle($parsed['shuffle']), //@TODO make normalizers configurable
+            $this->normalizeShuffle($parsed['shuffle']),
```
Comment (gabrielfs7) [inline model/import/Parser/CsvLineConverter.php:58]: We need to revert it

File: model/import/Parser/CsvLineConverter.php [hunk 4]
```diff
@@ -52,29 +55,24 @@ public function convert(array $line, TemplateInterface $template, $validationRep
...
 
-    private function hasValidationIssues(string $field, RecoverableLineValidationException $report): bool
-    {
-        return in_array($field, array_column($report->getWarnings(), 'field'));
-    }
-
     private function normalizeLanguage(string $language): string
     {
         $groups = explode('-', $language);
+
```
Comment (gabrielfs7) [inline model/import/Parser/CsvLineConverter.php:70]: Please, revert it

**Comment by gabrielfs7** [comment]
@habemuscode I did the backporting using proper commit hashes that we had before: https://github.com/oat-sa/extension-tao-itemqti/pull/1857/files

Please take a look

**Comment by habemuscode** [comment]
Closed in favor of https://github.com/oat-sa/extension-tao-itemqti/pull/1857

**Pull request closed without merging**